internal/core/accessibility.rs

```diff
@@ -24,6 +30,68 @@ pub enum AccessibleStringProperty {
     ValueStep,
 }
 
+// Defines an accessibility action.
+pub enum AccessibilityAction {
+    Default,
+    Focus,
+    Blur,
+    Collapse,
+    Expand,
+    CustomAction,
+    Decrement,
+    Increment,
+    // FIXME: Can it be currently supported?
+    HideToolTip,
+    // FIXME: Can it be currently supported?
+    ShowToolTip,
```

Comment:
I guess there is no point having action that don't map to action that we can define in the .slint language. And we should only support what we can implement.

Reply:
Agree. I have removed the unsupported ones.